identity-apps-core/apps/authentication-portal/src/main/webapp/smsOtp.jsp

```diff
@@ -158,10 +158,16 @@
                               } %>
 
                             <div class="field">
-                                <% if (request.getParameter("screenValue") != null) { %>
+                                <%
+                                    String screenValue = request.getParameter("screenValue");
+                                    if (screenValue == null) {
+                                        screenValue = request.getParameter("screenvalue");
+                                    }
```

Comment:
Is this correct ?
do we need this if logic ?

Reply:
Yea. Here it checks for `screenValue` first. If it's null checks for `screenvalue`. Note the case of letter `v`.

Reply:
Ahh cool!